src/tasks/batch_integration/metrics/cell_cycle_conservation/script.py

```diff
@@ -26,19 +25,19 @@
 
 print('compute score', flush=True)
 score = cell_cycle(
```

Comment:
Can a 'layer' argument be added to the `cell_cycle` function? This would save us the trouble of having to copy the normalized data to `.X`.

Reply:
This would be possible, I'd have to implement this in scib first though

Reply:
Ok! When the time comes, let's do this in a separate PR.